ui/templates/base_error.html

```diff
@@ -7,7 +7,7 @@
 {% block extrahead %}
```

Comment:
just noticed while i was making changes that this file used a mix of tabs and spaces. i changed it to spaces-only